wfsim/core.py

```diff
@@ -809,6 +831,8 @@ def get_truth(self, instruction, truth_buffer):
                 truth_buffer[ix]['t_first_{name}'.format(name=name)] = np.nan
                 truth_buffer[ix]['t_last_{name}'.format(name=name)] = np.nan
                 truth_buffer[ix]['t_sigma_{name}'.format(name=name)] = np.nan
+            truth_buffer[ix]['endtime'] = np.max(
```

Comment:
t_last_photon should be the endtime already, not sure why the electrons need to be added in here.

Reply:
Endtime is wanted by strax: https://strax.readthedocs.io/en/latest/advanced/plugin_dev.html
Adding this makes it possible to include the truth in a loopplugin making matching really easy